### ISSUE TITLE
Migrate to Hugo Modules fails becase 'go' can't be found

### ISSUE THREAD

**Issue by snunez1** [google/docsy#1062]
Is there a way to do this without installing 'go'?  I'm using the hugo binary on MS Windows and get:
```
Error: failed to init modules: exec: "go": executable file not found in %PATH%
```
at the step: ` hugo mod init github.com/me/my-existing-site`. It would be nice to workaround this and not have to install go just to run Docsy; go isn't used anywhere in the organisation.

**Comment by deining** [comment]
> Is there a way to do this without installing 'go'?

Yes, it is.

Please have a look at the assets of the [release page](https://github.com/google/docsy-example/releases) of the `docsy-example` project. There you will find an archive [`_vendor.zip`](https://github.com/google/docsy-example/releases/download/v0.2.0/_vendor.zip). Extract this archive in the root directory of your project. This should result in a new folder `_vendor` created in your project root folder. Run `hugo server` afterwards, and your site should be up and running. Let me know if this works for you.

**Comment by snunez1** [comment]
Thanks.  After a mighty battle with `git`, I managed to get submodules working. I had cloned from the example project, so my upstream/origin was not docsy, and I found git submodules quite fiddly. 

I'll wait until the next major upgrade to move to modules.

I can leave this open as a reminder to document this option (using _vendor.zip), or close it.  I think it would be nice to mention it in the section on upgrading Docsy.
